docker-create-debian-image

```diff
@@ -2,7 +2,20 @@
 
 set -e -o pipefail
 
-usage="Usage: $0 [-h] [-k] [-i <image name>] [-q] [-m <mirror>/debian/ubuntu...] [-t <variant>] [distribution [architecture]]"
+usage="Usage: ${0##*/} [-h] [-k] [-i <image name>] [-q] [-m <mirror>/debian/ubuntu...] [-t <variant>] [distribution [architecture]]
+
+  -h              - this help
+  -k              - keep the temporary directory for the root filesystem after build
+  -i <image name> - name of the Docker image (default: debian-${distribution}-${architecture})
```

Comment:
You should escape ${distribution} and ${architecture}, otherwise the shell will try to expand the variables, and they are not yet assigned.
Static analysis reference: https://travis-ci.com/lpenz/docker-debian-releases/jobs/196860350